go/logic/gtidset.go

```diff
@@ -0,0 +1,56 @@
+package logic
+
+import (
+	"strings"
+)
+
+type GTIDSet struct {
```

Comment:
Ooh TIL about `gtid_subtract` https://github.com/github/orchestrator/blob/19c0209e6790912909ff6849600251c0e2a439ab/go/inst/instance_topology_dao.go#L1164 (per @shlomi-noach's link)

Perhaps I don't need this logic after all.